packages/toggle-switch/toggle-switch.scss

```diff
@@ -0,0 +1,66 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+@import "~@fremtind/jkl-core/mixins/_utils.scss";
+@import "~@fremtind/jkl-core/_functions.scss";
+
+$toggle-width: rem(72px);
+$toggle-height: rem(40px);
+$toggle-focus-outline-color: $himmel;
+
+.jkl-toggle-switch {
+    & input {
+        opacity: 0;
+        width: 0;
+        height: 0;
+    }
+    position: relative;
+    min-width: $toggle-width * 5;
```

Comment:
This should not be needed. I have a small refactor that gets rid of this and cleans up a few other things

Reply:
Cool, yeah, this should be aviodable